### DISCUSSION TITLE
Double Click on bars to return to default (Little quality of life thing)

### DISCUSSION THREAD

**Discussion by KittenzExe** [ppy/osu#21532] [Ideas] [answered]
Little things like returning a bar to the default option or something like that would be nice. I know there is a little snap to go go the default but as a little quality of life it would be cool!

![image](https://user-images.githubusercontent.com/67358250/205969493-be866ba0-bc25-4c6b-8632-d8060e192021.png)
(An example with the playback bar, double click to return to 1.0)

**Comment by mk56-spn** [comment, marked as answer]
https://github.com/ppy/osu/discussions/21505 asked like 2 days ago, please search before asking!

**Reply by KittenzExe** [reply]
ah yea. just seen it. i skimmed through briefly. must have missed it. ty
